src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampRequestNetstandard21Wrapper.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+#if IS_SIGNING_SUPPORTED && IS_CORECLR
+using System.Net.Http;
+using System.Net.Http.Headers;
+#endif
```

Comment:
The class defined in this function is wrapped in the same `#if` `#endif` block. Why are only these using statements excluded when the entire class hidden?  Just have a single `#if` on line 3/4.

Reply:
Thanks. Will fix in #3571